src/strings/PowerFxResources.en-US.resx

```diff
@@ -5139,7 +5139,11 @@
     <comment>Error Message.</comment>
   </data>
   <data name="WrnDelagationTableNotSupported" xml:space="preserve">
-    <value>Delegating this operation on table '{0}' is not supported.</value>
+    <value>Delegation warning. Table '{0}' part might not work correctly.</value>
```

Comment:
>Delegation warning [](http://example.com/codeflow?start=11&length=18)

The error type for this error is already a warning - we probably don't need to say it again in the message.